packages/chains/deployments/bsc.json

```diff
@@ -33766,4 +33766,4 @@
       ]
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
🙄

Reply:
I saw that just before I pushed the commits... I will refactor the deploy script to use the new config